charts/access/slack/values.schema.json

```diff
@@ -0,0 +1,373 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "http://example.com/example.json",
+    "default": {},
+    "required": [
+        "image",
+        "imagePullSecrets",
+        "nameOverride",
+        "fullnameOverride",
+        "podAnnotations",
+        "podSecurityContext",
+        "securityContext",
+        "nodeSelector",
+        "tolerations",
+        "affinity",
+        "teleport",
+        "slack",
+        "roleToRecipients",
+        "log"
+    ],
+    "properties": {
+        "image": {
+            "$id": "#/properties/image",
+            "type": "object",
+            "default": {},
+            "examples": [
+                {
+                    "repository": "146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-plugin-slack",
```

Comment:
This should be updated to quay too.

Reply:
https://github.com/gravitational/teleport-plugins/pull/494/commits/a4f6e29685c1ab55a4cfef5d44eda7d1fda9d02e